LayoutTests/fast/canvas/canvas-strokePath-round-lineCap.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script>
+    async function doLoad() {
+      if (window.testRunner) {
+        testRunner.waitUntilDone();
+        if (testRunner.dontForceRepaint)
+          testRunner.dontForceRepaint();
+      }
+
+      var canvas = document.createElement('canvas');
+      document.body.appendChild(canvas);
+      var ctx = canvas.getContext('2d');
+
+      canvas.width = 200;
+      canvas.height = 200;
+      ctx.lineWidth = 29.04;
+      ctx.lineCap = "round";
+
+      ctx.clearRect(0, 0, 200, 200);
+
+      await new Promise(requestAnimationFrame);
+      await new Promise(requestAnimationFrame);
+
+      ctx.strokeStyle = "#f00";
+      ctx.beginPath();
+      ctx.moveTo(50, 50);
+      ctx.lineTo(100, 150);
+      ctx.stroke();
+
+      ctx.strokeStyle = "#0f0";
+      ctx.beginPath();
+      ctx.moveTo(100, 150);
+      ctx.bezierCurveTo(150, 100, 150, 100, 150, 50);
+      ctx.stroke();
+
+      await new Promise(requestAnimationFrame);
```

Comment:
I think you only need these two, not the two earlier ones.

BTW tests using `testRunner.dontForceRepaint()` only really work in WK2, but if they don't fail in WK1 I guess it's OK.

Reply:
If I remove the first pair of rAFs, the test still works fine with the patch, but then it also works *without* the patch, so it's not as good at catching potential regressions.
My guess is that:
- Without these first rAFs, the **whole** canvas gets drawn, so the smaller strokes don't get clipped;
- These first rAFs commit the `clearRect` to the screen, and then when we add strokes, the display refresh is limited to the incorrect-without-the-patch stroke bounds, so they get clipped.

Looking at the EWS logs, the test runs and passes on WK1; and I've verified locally that it does fail without the patch.